Loading libgnc-gnome hangs; not sure why, but this initialization 
doesn't appear to be necessary.

--- origsrc/gnucash-2.4.0/src/import-export/qif-import/qif-import.scm	2010-12-21 20:01:10.000000000 -0600
+++ src/gnucash-2.4.0/src/import-export/qif-import/qif-import.scm	2011-01-27 02:35:14.529242900 -0600
@@ -9,8 +9,8 @@
 (use-modules (gnucash main)) ;; FIXME: delete after we finish modularizing.
 
 ;; We do this initialization here because src/gnome isn't a real module.
-(load-extension "libgnc-gnome" "scm_init_sw_gnome_module")
-(use-modules (sw_gnome))
+;; (load-extension "libgnc-gnome" "scm_init_sw_gnome_module")
+;; (use-modules (sw_gnome))
 
 (use-modules (gnucash gnc-module))
 (use-modules (ice-9 regex))
